### ISSUE TITLE
How the reproduction of the results of the model achieves the results in the paper

### ISSUE THREAD

**Issue by GuanghuiWei** [noahzn/Lite-Mono#141]
Hello, thanks for great code.I got the pretrained model and I ran the train.py to reproduce the lite-mono, I trained with the same dependencies on the A6000，but I got different results.
![564](https://github.com/noahzn/Lite-Mono/assets/119914305/10fdb2af-e701-487e-b5cb-a917f883b5f6)

**Comment by Wshenv** [comment]
> Hello, thanks for great code.I got the pretrained model and I ran the train.py to reproduce the lite-mono, I trained with the same dependencies on the A6000，but I got different results.

![Uploading 564.png…]()

**Comment by noahzn** [comment]
Hi, please make sure you use the same versions of dependencies as we used. There are also some tickets discussing the reproduction of the results. Please search in this repo.

**Comment by GuanghuiWei** [comment]
Thanks for your suggestion, I reconfigured the dependencies again and made the data set in.jpg format according to the way of monodepth2, but the result of retraining is still not satisfactory. This is my dependency, and only the version of tensorboardX dependency is inconsistent, but I think it will not affect the result of training
![1](https://github.com/noahzn/Lite-Mono/assets/119914305/03817207-d27e-48d0-b971-319305dcef62)
![2](https://github.com/noahzn/Lite-Mono/assets/119914305/89dc0e11-5195-40f3-81e2-4e8638f2dfca)

**Comment by GuanghuiWei** [comment]
In addition, I found a torch>=1.8 in requirements.txt of linear_warmup_cosine_annealing_warm_restarts_weight_decay=1.0. Is that the reason?

**Comment by noahzn** [comment]
It should be fine because we could reproduce our results using pytorch 1.7.1 and 1.12.1
https://github.com/noahzn/Lite-Mono/issues/58

**Comment by GuanghuiWei** [comment]
Thank you very much. I tried to adjust the dropout to 0.3 and conducted the experiment again, and got a result very close to the paper in the 24th round. I have another question for you. Is the data set in jpg format converted strictly according to the conversion command in monodepth2?
![Figure_1](https://github.com/noahzn/Lite-Mono/assets/119914305/7a9749d9-7e80-40d3-910c-e96e61ac04ae)
![Figure_2](https://github.com/noahzn/Lite-Mono/assets/119914305/c6ac3aca-335b-469c-9ac7-472947c111f7)

**Comment by noahzn** [comment]
Hi, good to know that you have better results now! Yes, we directly converted the images to .jpg according to the command in the monodepth2's repo.
